modules/serverless-creating-a-kafka-event-sink.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-event-sinks.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-creating-a-Kafka-event-sink_{context}"]
+
+= Creating a Kafka event sink
+
+As a developer, you can create an event sink to receive events from a particular source, such as a bootstrap server and send them to a kafka topic. 
+
+.Prerequisites
+* You have installed the Red Hat OpenShift Serverless operator, with Knative Serving, Knative Eventing, and Knative Kafka APIs, from the Operator Hub.
+
+* Your cluster administrator has added the KafkaSink resource, provided by Red Hat, in the *Developer Catalog*.
```

Comment:
> * Your cluster administrator has added the KafkaSink resource, provided by Red Hat, in the *Developer Catalog*.

what does this mean in practice? isn't installing Knative enough to get it in the catalog?

Reply:
cc @invincibleJai

Reply:
@invincibleJai @vikram-raj, Should we remove this prerequisite point?

Reply:
Yes, we can remove this. Setting up knative with Knative Kafka is enough. It will show the KafkaSink in the catalog.